sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/DocumentEntity.java

```diff
@@ -58,7 +58,7 @@ public String getCategory() {
      * @param category the category value to set.
      * @return the DocumentEntity object itself.
      */
-    void setCategory(String category) {
+    private void setCategory(String category) {
```

Comment:
Not familiar with the code in Java, but shouldn't `DocumentEntity`, `DocumentFootnote`, and `DocumentCaption` be removed?

Reply:
good point, I think the models were still around, deleted them, thanks!